pkg/api/v1/status/alert_configurations.go

```diff
@@ -0,0 +1,292 @@
+	EmailEnabled *bool `json:"emailEnabled,omitempty"`
+	// The Flowdock personal API token. Populated for the FLOWDOCK notifications type. If the token later becomes invalid, Atlas sends an email to the project owner and eventually removes the token.
+	FlowdockAPIToken string `json:"flowdockApiToken,omitempty"`
+	// Flowdock flow namse in lower-case letters.
+	FlowName string `json:"flowName,omitempty"`
+	// Number of minutes to wait between successive notifications for unacknowledged alerts that are not resolved.
+	IntervalMin int `json:"intervalMin,omitempty"`
+	// Mobile number to which alert notifications are sent. Populated for the SMS notifications type.
+	MobileNumber string `json:"mobileNumber,omitempty"`
+	// Opsgenie API Key. Populated for the OPS_GENIE notifications type. If the key later becomes invalid, Atlas sends an email to the project owner and eventually removes the token.
+	OpsGenieAPIKey string `json:"opsGenieApiKey,omitempty"`
+	// Region that indicates which API URL to use.
+	OpsGenieRegion string `json:"opsGenieRegion,omitempty"`
+	// Flowdock organization name in lower-case letters. This is the name that appears after www.flowdock.com/app/ in the URL string. Populated for the FLOWDOCK notifications type.
+	OrgName string `json:"orgName,omitempty"`
+	// PagerDuty service key. Populated for the PAGER_DUTY notifications type. If the key later becomes invalid, Atlas sends an email to the project owner and eventually removes the key.
+	ServiceKey string `json:"serviceKey,omitempty"`
+	// Flag indicating if text message notifications should be sent. Populated for ORG, GROUP, and USER notifications types.
+	SMSEnabled *bool `json:"smsEnabled,omitempty"`
+	// Unique identifier of a team.
+	TeamID string `json:"teamId,omitempty"`
+	// Label for the team that receives this notification.
+	TeamName string `json:"teamName,omitempty"`
+	// Type of alert notification.
+	TypeName string `json:"typeName,omitempty"`
+	// Name of the Atlas user to which to send notifications. Only a user in the project that owns the alert configuration is allowed here. Populated for the USER notifications type.
+	Username string `json:"username,omitempty"`
+	// VictorOps API key. Populated for the VICTOR_OPS notifications type. If the key later becomes invalid, Atlas sends an email to the project owner and eventually removes the key.
+	VictorOpsAPIKey string `json:"victorOpsApiKey,omitempty"`
+	// VictorOps routing key. Populated for the VICTOR_OPS notifications type. If the key later becomes invalid, Atlas sends an email to the project owner and eventually removes the key.
+	VictorOpsRoutingKey string `json:"victorOpsRoutingKey,omitempty"`
+	// The following roles grant privileges within a project.
+	Roles []string `json:"roles,omitempty"`
+}
+
+func NotificationFromAtlas(notification mongodbatlas.Notification) Notification {
+	return Notification{
```

Comment:
Why not use `JSONCopy` here?
```
result := & status.Notification{}
err := compat.JSONCopy(result, s)
return result, err
```

Reply:
Then this error will have to be returned and handled somehow. And the current approach makes it so that the Operator can put any structure from the Altas library into our structure.

Reply:
But this is a better long-term solution as no changes will have to be made in case of the changes in structs, assuming these structs are also not hardcoded ofc.

And it's also much shorter even with error handling :)

Reply:
I don't think so. If this structure changes in the Atlas lib, then we still have to add new fields to our structure. 
BTW after each bump of Atlas lib, we have to check if some of the JSON tags change. So it doesn't look good for long-term support.